tests/e2e/util.go

```diff
@@ -941,6 +941,44 @@ func updateCSIDeploymentTemplateFullSyncInterval(client clientset.Interface, min
 	return oldValue
 }
 
+// updateCSIDeploymentProvisionerTimeout helps to update the timeout value
+// in deployment template. For this to take effect,
+func updateCSIDeploymentProvisionerTimeout(client clientset.Interface, namespace string, timeout string) {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+	ignoreLabels := make(map[string]string)
+	list_of_pods, err := fpod.GetPodsInNamespace(client, csiSystemNamespace, ignoreLabels)
+	gomega.Expect(err).NotTo(gomega.HaveOccurred())
+	num_csi_pods := len(list_of_pods)
+
+	deployment, err := client.AppsV1().Deployments(namespace).Get(
+		ctx, vSphereCSIControllerPodNamePrefix, metav1.GetOptions{})
+	gomega.Expect(err).NotTo(gomega.HaveOccurred())
+	containers := deployment.Spec.Template.Spec.Containers
+	for _, c := range containers {
+		if c.Name == "csi-provisioner" {
+			for i, e := range c.Args {
+				if strings.Contains(e, "timeout") {
+					if strings.Contains(e, timeout) {
```

Comment:
this is not the right check, if some wants to use '30s' as timeout value when default value is '300s', this check will pass

Reply:
Corrected the check to match the whole string